lang/russian.json

```diff
@@ -150,319 +150,319 @@
     "imageSlideshowBlockSpaceBetween": "Пространство между изображениями в пикселях",
     "imageSlideshowBlockTitle": "Слайд-шоу изображений",
     "imageTool": "Изображение",
-    "import": "Импорт",
-    "importFailedError": "Импорт файла не удался!",
+    "import": "Импортировать",
+    "importFailedError": "Не удалось импортировать файл!",
     "importFromUrl": "Импорт",
     "importUrl": "Введите URL-адрес файла",
-    "importing": "Импорт ...",
-    "inPageEdit": "Режим внутристраничного редактирования",
+    "importing": "Импортируем ...",
+    "inPageEdit": "Редактировать прямо на странице",
     "indent": "Отступ",
     "inlineCode": "Встроенный код",
     "insertCodeBlock": "Вставить блок кода",
     "insertHorizontalLine": "Горизонтальная линия",
-    "insertImage": "Вставка изображения",
+    "insertImage": "Вставить изображение",
     "insertLink": "Вставить ссылку",
-    "insertTable": "Вставка таблицы",
+    "insertTable": "Вставить таблицу",
     "invalidEmailError": "Неверный адрес электронной почты!",
     "invalidFieldError": "Недействительное поле",
-    "invalidFormError": "Неполные или неправильные данные!<br />Все поля обязательны для заполнения и должны быть заполнены в соответствии с описанием!",
-    "invalidImageError": "Предоставлено недействительное изображение.",
+    "invalidFormError": "Неполные или неверные данные!<br />Все поля обязательны для заполнения и должны быть заполнены в соответствии с описанием!",
+    "invalidImageError": "Предоставлено неверное изображение.",
     "invalidUsernameError": "Неверное имя пользователя! Имя пользователя может содержать только следующие символы:",
     "isNotImage": "не является изображением",
-    "italic": "Italic",
-    "jumpbarButtonText": "Поиск или открытие",
-    "keepPagePrivate": "Сохраняй конфиденциальность страницы",
-    "landscape": "Пейзаж",
-    "large": "Большой",
+    "italic": "Курсив",
+    "jumpbarButtonText": "Найти или открыть",
+    "keepPagePrivate": "Сохранить страницу приватной",
+    "landscape": "Альбомная ориентация",
+    "large": "Огромный",
     "lastPublished": "Последняя публикация",
-    "latestActivity": "Последняя активность",
+    "latestActivity": "Последняя активность:",
     "layoutDefault": "Автоматическая ширина",
     "layoutStretch": "Растягивающийся блок",
     "layoutWidth": "Ширина",
-    "letterSpacing": "Расстановка букв",
-    "lineHeight": "Высота строки",
-    "lineTool": "Строка",
+    "letterSpacing": "Межбуквенный интервал",
+    "lineHeight": "Высота линии",
+    "lineTool": "Линия",
     "link": "Ссылка",
     "list": "Список",
-    "loading": "Loading...",
-    "mailBlockAddressFieldLabel": "Ярлык для адресного поля",
```

Comment:
I am not sure if this is clear enough in the english file. The field labels should basically guide the user to add a custom label that is shown above the address, body or subject fields of the generated mail form.